src/App.tsx

```diff
@@ -12,9 +12,15 @@ import { PCDetails } from "./pages/Products/PC/PCDetails";
 import { PCs } from "./pages/Products/PC/PCs";
 
 export const App = () => {
+  const [userRole, setUserRole] = useState("Customer");
+
+  const handleUserRoleChange = (role: string) => {
+    setUserRole(role);
+  };
+
   return (
     <div className="m-2">
-      <Navbar />
+      <Navbar onUserRoleChange={handleUserRoleChange} userRole={userRole} />
```

Comment:
To są takie propsy do komponentów, to takie coś jak parametry w metodach, tu jest obsługa roli i przycisku `Add new product`